### DISCUSSION TITLE
dataset used for making cyclonic and anticyclonic files

### DISCUSSION THREAD

**Discussion by devyani2509** [AntSimi/py-eddy-tracker#175] [General]
Hello sir,

can we use monthly SSH reanalysis dataset such as ORAS4 ?

**Comment by AntSimi** [comment]
I never used this type of data (monthly sampled) for eddy detection.
Code will produce output but :
If it's a monthly mean => mesoscale will be smoothed in high wariability region.
Tracking will be complex&bad with only observation by month, one by week, it's already hard.

I am interested to the results, if you try it
